library/Director/Objects/IcingaObjectGroups.php

```diff
@@ -183,6 +183,10 @@ public function add($group, $onError = 'fail')
             return $this;
         }
 
+         if (is_int($group)) {
+             $group = "$group";
```

Comment:
Please use `$group = (int) $group;` instead of `"$group"`

Reply:
this is what I am trying to avoid - `$group` already is an integer at this point, hence i could cast it to string if you don't like the quotations....

Reply:
My bad, this should of course have read "Please use `$group = (string) $group;`"

Reply:
to keep the footprint small I amended the change to the existing commit (now [917d62a](https://github.com/Icinga/icingaweb2-module-director/pull/2822/commits/917d62ad5aab5b954132095cd5f3cd6affe26c5d))